openshift-container-platform-4/policies/CA-Security_Assessment_and_Authorization/component.yaml

```diff
@@ -302,18 +302,16 @@
 - control_key: CA-9
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: partial
+  implementation_status: not applicable
   narrative:
     - key: a
       text: |
-        Engineering progress can be tracked at:
-
-        https://issues.redhat.com/browse/CMP-256
+        CA-9 is an organizational control outside the scope
```

Comment:
```suggestion
        Organizational processes to authorize internal connections of
        organization-defined information system components or classes of
        components to the information system is an organizational control outside the scope
```

Reply:
To be consistent, and address user feedback, ``not applicable`` control responses should be more verbose and include text from the NIST control.